test/unittest/transport/can/test_can_delegate.cpp

```diff
@@ -3,18 +3,32 @@
 /// Copyright Amazon.com Inc. or its affiliates.
 /// SPDX-License-Identifier: MIT
 
-#include <libcyphal/transport/can/delegate.hpp>
-
 #include "../../memory_resource_mock.hpp"
 #include "../../tracking_memory_resource.hpp"
 
-#include <numeric>
+#include <canard.h>
+#include <cetl/pf17/cetlpf.hpp>
+#include <libcyphal/transport/can/delegate.hpp>
+#include <libcyphal/transport/errors.hpp>
+#include <libcyphal/transport/types.hpp>
+#include <libcyphal/types.hpp>
+
 #include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <array>
+#include <cstddef>
+#include <cstdint>
+#include <numeric>
+#include <string>
+#include <utility>
+#include <vector>
 
 namespace
 {
-using namespace libcyphal::transport;
-using namespace libcyphal::transport::can;
+
+using libcyphal::TimePoint;
+using namespace libcyphal::transport;  // NOLINT This our main concern here in the unit tests.
```

Comment:
It is best to provide NOLINT comments with a specific rule to be suppressed. CLion can inject these automatically.